packages/radio/src/mwc-radio-base.ts

```diff
@@ -136,16 +142,13 @@ export class RadioBase extends FormElement {
 
   focus() {
     this.focusNative();
+    this.handleRippleFocus();
```

Comment:
I assume this doesn't need to be added to a blur fn because blur is implicit in a radio group?

Reply:
I just realized that this would not work in all cases and wondered why I didn't noticed that in my tests, I see now that it `mdc-radio` has its own focus styling `.mdc-radio .mdc-radio__background::before` & `.mdc-radio__native-control:focus + .mdc-radio__background::before`

Is this something that should be removed and replaced by the ripple or should we not use the ripple for focus?

Reply:
The current styling doesn't match material, as focus is always in the accent color.
![image](https://user-images.githubusercontent.com/5662298/88095827-02c1a700-cb96-11ea-8584-ab42716d039e.png)

Reply:
Hmm I'm going to have to pass this onto @dfreedm since he's been working with radio more-closely

Reply:
I believe that styling is holdover from MDC for a CSS-only ripple. You should be able to turn it off by adding a class named `mdc-radio--upgraded` to the `mdcRoot` node

Reply:
Hmm, radio works a bit differently than checkbox, the right classname is actually `mdc-ripple-upgraded`

Reply:
That doesn't seem to work, but this does:
```scss
  .mdc-radio__background::before {
    content: none;
  }
```